src/VisualStudio/CSharp/Impl/EventHookup/EventHookupSessionManager.cs

```diff
@@ -1,26 +1,34 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.ComponentModel.Composition;
 using System.Linq;
 using System.Threading;
-using System.Windows;
+using Microsoft.CodeAnalysis.Classification;
 using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
 using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Adornments;
 using Microsoft.VisualStudio.Text.Editor;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.EventHookup
 {
+    [Export]
     internal sealed partial class EventHookupSessionManager : ForegroundThreadAffinitizedObject
     {
+        private IToolTipService _toolTipService = null;
+        private IToolTipPresenter _toolTipPresenter;
+
         internal EventHookupSession CurrentSession { get; set; }
 
         // For test purposes only!
-        internal FrameworkElement TEST_MostRecentQuickInfoContent { get; set; }
+        internal ClassifiedTextElement TEST_MostRecentToolTipContent { get; set; }
 
-        internal EventHookupSessionManager()
+        [ImportingConstructor]
+        internal EventHookupSessionManager([Import] IToolTipService toolTipService)
```

Comment:
You don't need to say [Import] in an importing constructor unless you're providing extra things.

Reply:
removed